automated-testing/ui-testing/readme.md

```diff
@@ -0,0 +1,64 @@
+# UI Testing
+
+This section is primarily geared towards web-based UIs, but the guidance is similar for mobile and OS based applications.  
+
+## Goals
+
+UI testing provides the ability to ensure that users have a consistent visual user experience across a variety of means of access and that the user interaction is consistent with the function requirements.
+
+### UI tests have several goals
+
+- Ensure the UI appearance and interaction satisfy the functional and non-functional requirements
+- Detect changes in the UI both across devices and delivery platforms and between code changes
+- Provide confidence to designers and developers the user experience is consistent
+- Support fast code evolution and refactoring while reducing the risk of regressions
```

Comment:
Would be good to add a note about accessibility as a goal here - very important for UI tests.

Reply:
Yeah I wasn't entirely sure how much to feature that, I believe there is another TBD section of the playbook that deals with accessibility testing, and there are tools that can evaluate the accessibility of a site as a pipeline task I believe, so it deserves an entire dedicated section.   That said, I think one could definitely write UI tests with pure keyboard commands to see if the site is navigable with just the keyboard, which could be very interesting, so I'll update some language here.